_includes/manuals/2.0/3.1.1_supported_platforms.md

```diff
@@ -8,16 +8,15 @@ The following operating systems are supported by the installer, have packages an
     * check the above repositories because the command can silently fail when subscription does not provide it: `yum repolist`
   * Apply all SELinux-related errata.
 * CentOS, Scientific Linux or Oracle Linux 7
+  * **Note:** The CentOS packages are not tested on Scientific Linux or Oracle Linux. The Foreman installation on Scientific Linux or Oracle Linux may or may not work.
   * Architectures: x86_64 only
   * [EPEL](http://fedoraproject.org/wiki/EPEL/FAQ#How_can_I_install_the_packages_from_the_EPEL_software_repository.3F) is required
 * Ubuntu 18.04 (Bionic)
   * Architectures: i386, amd64, aarch64
 * Debian 10 (Buster)
   * Architectures: i386, amd64, aarch64
 
-It is assumed to use the PostgreSQL version provided by the OS, which is currently:
-
-* PostgreSQL 9.2 or newer
+PostgreSQL version 10 or newer. For EL 7 this is available from Software Collections.
```

Comment:
Are we testing on PostgreSQL 10? I Think we were going with 12+

Reply:
https://github.com/theforeman/theforeman.org/pull/1550#discussion_r404848830